src/backend/distributed/utils/shard_utils.c

```diff
@@ -39,20 +42,44 @@ GetTableLocalShardOid(Oid citusTableOid, uint64 shardId)
 
 
 /*
- * GetReferenceTableLocalShardOid returns OID of the local shard of given
- * reference table. Caller of this function must ensure that referenceTableOid
- * is owned by a reference table.
+ * GetNoneDistTableLocalShardRelationId returns relation id for local shard of
+ * DISTRIBUTE_BY_NONE table with noneDistTableId. Caller of this function must
+ * ensure that and input relation has a local shard placement in current node.
  */
 Oid
-GetReferenceTableLocalShardOid(Oid referenceTableOid)
+GetNoneDistTableLocalShardRelationId(Oid noneDistTableId)
```

Comment:
> can we find an alternative that is not too long?
To me, it doesn't seem to be much possible :/ 

I also think that it is obvious that NoneDistTable = DISTRIBUTE_BY_NONE table ?

Reply:
stepping back a bit, to me it seems like the only reason we use DISTRIBUTE_BY_NONE etc is to understand table types, and if that is the case, it seems more readable to reflect this in the code. So whenever I see `DISTRIBUTE_BY_NONE` I convert it to something like this in my mind `Okay, this is either a citus local table or reference table`. Because we are the ones who is creating this, we say that `citus local table` and `reference tables` will have `DISTRIBUTE_BY_NONE` as their replication model. So to me, it seems like the only place that `DISTRIBUTE_BY_NONE` is useful and necessary is when deciding if a table is `citus local table` or a `reference table`. I am not quite getting why we need to use it in other places, can you elaborate that please?

Reply:
Realized that this function is not used anymore, removing it :)